scenarioo-client/app/components/detailarea/detailarea.component.html

```diff
@@ -15,41 +15,83 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
-<div class="sc-meta-data-panel row">
+<div class="details-area-container">
 
-    <button (click)="togglePannelCollapsedValue()" [attr.aria-expanded]="!isPanelCollapsed" aria-controls="collapseBasic"
-            class="btn btn-primary pull-right" id="sc-showHideDetailsButton" type="button">
-        {{ isPanelCollapsed ? "Show details" : "Hide details" }}
-    </button>
-
-    <div [ngClass]="isPanelCollapsed ? 'col-sm-11' : 'col-sm-8'">
-        <ng-content></ng-content>
+    <!-- Top Level Button Row -->
+    <div class="row">
+        <button (click)="togglePannelCollapsedValue()" [attr.aria-expanded]="!isPanelCollapsed" aria-controls="collapseBasic"
+                class="btn btn-primary pull-right" id="sc-showHideDetailsButton" type="button">
+            {{ isPanelCollapsed ? "Show details" : "Hide details" }}
+        </button>
     </div>
 
-    <div #metaDataPanel class="detail-container pull-right"  [ngClass]="isPanelCollapsed ? '' : 'col-sm-4'">
-        <div class="detail-row">
-            <div (click)="togglePannelCollapsedValue()" class="detail-cell detail-button">
-                <span *ngIf="!isPanelCollapsed">
-                    <i class="fas arrowCenter fa-angle-right"></i>
-                </span>
-                <span *ngIf="isPanelCollapsed">
-                    <i class="fas arrowCenter fa-angle-left"></i>
-                </span>
-            </div>
+    <!-- Content Area -->
+    <div class="row flex">
+
+
+        <!-- Page Content left (will fill if details are collapsed) -->
+        <div class="detailarea-page-content"
+             [ngClass]="isPanelCollapsed ? 'col-sm-11 flex-fill' : 'col-sm-8'">
```

Comment:
As an alternative to _'col-sm-11 flex-fill'_: _col-sm-12_, this causes the detailarea to take up the whole width (bootstrap's grid system has 12 columns)